cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -43,22 +44,23 @@ const errorMessageKey = "errorMessage"
 type CosmosSub struct {
 	TmProvider              string
 	EthProvider             string
-	RegistryContractAddress common.Address
 	PrivateKey              *ecdsa.PrivateKey
 	DB                      *leveldb.DB
 	SugaredLogger           *zap.SugaredLogger
+	NetworkID               oracletypes.NetworkID
```

Comment:
We should update the NetworkID code to use the new NetworkDescriptor enum that's in future/peggy2 now:

```
commit a255327cb87d16cbbf7ca04b7689c9260fad3b81 (origin/future/peggy2, future/peggy2)
Merge: 88c24f35 e50aa0fc
Author: James Moore <banshee@banshee.com>
Date:   Mon Jun 21 10:26:01 2021 -0700

    Merge pull request #1435 from Sifchain/future/peggy2-networkdescriptors
    
    Added potential protobuf file for network descriptors
```

Reply:
I can see the conflict, will pull your code change.